spec/models/list_item_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe ListItem do
+  describe "#display_title" do
+    it "shows the rummager title by default" do
+      stub_request(:get, RummagerHelper::SEARCH_ENDPOINT)
```

Comment:
What is a test helper and how would I check? The term is not easily googleable - it sounds like a term of art in Ruby or Rails world?